pkg/commands/compute/language_rust.go

```diff
@@ -312,32 +319,57 @@ func (r *Rust) toolchainConstraint() {
 	// #nosec
 	// nosemgrep
 	cmd := exec.Command(args[0], args[1:]...)
-	stdoutStderr, err := cmd.CombinedOutput()
-	output := string(stdoutStderr)
+	stdout, err := cmd.Output()
+	output := string(stdout)
 	if err != nil {
-		return
+		return nil
 	}
 
 	versionPattern := regexp.MustCompile(`cargo (?P<version>\d[^\s]+)`)
 	match := versionPattern.FindStringSubmatch(output)
 	if len(match) < 2 { // We expect a pattern with one capture group.
-		return
+		return nil
 	}
 	version := match[1]
 
 	v, err := semver.NewVersion(version)
 	if err != nil {
-		return
+		return nil
 	}
 
 	c, err := semver.NewConstraint(r.config.ToolchainConstraint)
 	if err != nil {
-		return
+		return nil
 	}
 
 	if !c.Check(v) {
 		text.Warning(r.output, "The Rust version '%s' didn't meet the constraint '%s'\n\n", version, r.config.ToolchainConstraint)
 	}
+
+	return v
+}
+
+func (r *Rust) checkCargoConfigFileName(rustVersion *semver.Version) error {
+	dir, err := os.Getwd()
+	if err != nil {
+		r.errlog.Add(err)
+		return fmt.Errorf("getting current working directory: %w", err)
+	}
+
+	if !filesystem.FileExists(filepath.Join(dir, ".cargo", "config")) {
+		return nil
+	}
+
+	text.Warning(r.output, "The Cargo configuration file name is .cargo/config")
```

Comment:
Let's get some breathing space between the WARNING and the preceding line (as I can see from your example output in the PR description that they're butted up against each other)
```suggestion
	text.Warning(r.output, "\nThe Cargo configuration file name is .cargo/config")
```